### DISCUSSION TITLE
script/template toggle

### DISCUSSION THREAD

**Discussion by alcrabie** [vuejs/language-tools#1719] [Ideas]
hi ,i dont know if it is the right place to ask for this,
what i am loking for
is like what we see in framework documentation, a buttons groups to navigate from a html runder to the code,
but in our case a groups of button to navigate between template and script part / css

**Comment by johnsoncodehk** [comment]
You can use https://code.visualstudio.com/docs/editor/editingevolved#_go-to-symbol with Volar.
